### PR TITLE
Fix bitrate in log bis

### PR DESCRIPTION
**Pull request by Sami32**
I guess that i never saw the end of the lines...

https://github.com/UniversalMediaServer/UniversalMediaServer/issues/1150

### CODE REVIEW

File: src/main/java/net/pms/encoders/FFMpegVideo.java [hunk 1]
```diff
@@ -431,10 +431,9 @@ public FFMpegVideo(PmsConfiguration configuration) {
 
 		// Give priority to the renderer's maximum bitrate setting over the user's setting
 		if (rendererMaxBitrates[0] > 0 && rendererMaxBitrates[0] < defaultMaxBitrates[0]) {
-			defaultMaxBitrates = rendererMaxBitrates;
-			LOGGER.trace("Using the video bitrate limit from the renderer config (" + rendererMaxBitrates[0] + "Mb/s) which is lower than the one from the program settings (" + defaultMaxBitrates[0] + ")");
+			LOGGER.trace("Using the video bitrate limit from the renderer config (" + rendererMaxBitrates[0] + " Mb/s) which is lower than the one from the program settings (" + defaultMaxBitrates[0] + " Mb/s)");
```
Comment (Nadahar) [inline src/main/java/net/pms/encoders/FFMpegVideo.java]: @Sami32 Where did ```defaultMaxBitrates = rendererMaxBitrates;``` go? Moving it after the logger is ok, but removing it is not. That will probably mean something won't work as expected, ie. a bug.

File: src/main/java/net/pms/encoders/MEncoderVideo.java [hunk 2]
```diff
@@ -693,10 +693,9 @@ private String addMaximumBitrateConstraints(String encodeSettings, DLNAMediaInfo
 
 		// Give priority to the renderer's maximum bitrate setting over the user's setting
 		if (rendererMaxBitrates[0] > 0 && rendererMaxBitrates[0] < defaultMaxBitrates[0]) {
-			defaultMaxBitrates = rendererMaxBitrates;
-			LOGGER.trace("Using the video bitrate limit from the renderer config (" + rendererMaxBitrates[0] + "Mb/s) which is lower than the one from the program settings (" + defaultMaxBitrates[0] + ")");
+			LOGGER.trace("Using the video bitrate limit from the renderer config (" + rendererMaxBitrates[0] + " Mb/s) which is lower than the one from the program settings (" + defaultMaxBitrates[0] + " Mb/s)");
```
Comment (Nadahar) [inline src/main/java/net/pms/encoders/MEncoderVideo.java]: The same applies here.

File: src/main/java/net/pms/encoders/FFMpegVideo.java (hunk 1 above)
Comment (Sami32) [inline src/main/java/net/pms/encoders/FFMpegVideo.java]: @Nadahar It's why it's labelled as not ready ;-)
The complement was done by @valib in this PR: https://github.com/UniversalMediaServer/UniversalMediaServer/pull/1152/files

Thanks for reviewing.

File: src/main/java/net/pms/encoders/MEncoderVideo.java (hunk 2 above)
Comment (Sami32) [inline src/main/java/net/pms/encoders/MEncoderVideo.java]: The same applies here ;-)

File: src/main/java/net/pms/encoders/FFMpegVideo.java (hunk 1 above)
Comment (Sami32) [inline src/main/java/net/pms/encoders/FFMpegVideo.java]: @Nadahar Done.

**Comment by Sami32** [comment]
@Nadahar Done.

**Comment by Nadahar** [comment]
@Sami32 I rebased it and changed the syntax to how the "modern" logger works. There's lots of that old form around, but I always update log lines to the newer format when I touch them.

The reason to write it this way is:
* It's much easier to see how the text is going to be formatted (spaces and all)
* It's cheaper when the log mode is such that this line won't be logged. With the "old" (using ```+```) way of writing it, Java will have to concenate the strings (which is very expensive) before the evaluation of the log level is done. Using ```{}``` the concenation will take place in the logger instead, which means that it will only be done when that line as actually being logged. That way all those ```TRACE``` lines won't slow down the code that much when running without ```TRACE```.

**Pull request merged**